README.md

```diff
@@ -131,6 +131,39 @@ runtime:
     newgauge: "http.server.connstate.new.gauge"
     # (string) Name of the counter metric tracking new clients.
     newcounter: "http.server.connstate.new"
+  expvar:
+    # (string) Name of the metric tracking allocated bytes
+    alloc: "go_expvar.memstats.alloc"
+    # (string) Name of the metric tracking number of frees
+    frees: "go_expvar.memstats.frees"
+    # (string) Name of the metric tracking allocated bytes
+    heapalloc: "go_expvar.memstats.heap_alloc"
+    # (string) Name of the metric tracking bytes in unused spans
+    heapidle: "go_expvar.memstats.heap_idle"
+    # (string) Name of the metric tracking bytes in in-use spans
+    heapinuse: "go_expvar.memstats.heap_inuse"
+    # (string) Name of the metric tracking total number of object allocated"
+    heapobject: "go_expvar.memstats.heap_objects"
+    # (string) Name of the metric tracking bytes realeased to the OS
+    heaprealeased: "go_expvar.memstats.heap_released"
+    # (string) Name of the metric tracking bytes obtained from the system
+    heapstats: "go_expvar.memstats.heap_sys"
+    # (string) Name of the metric tracking number of pointer lookups
+    lookups: "go_expvar.memstats.lookups"
+    # (string) Name of the metric tracking number of mallocs
+    mallocs: "go_expvar.memstats.mallocs"
+    # (string) Name of the metric tracking number of garbage collections
+    numgc: "go_expvar.memstats.num_gc"
+    # (string) Name of the metric tracking duration of GC pauses
+    pausens: "go_expvar.memstats.pause_ns"
+    # (string) Name of the metric tracking total GC pause duration over lifetime process
+    pausetotalns: "go_expvar.memstats.pause_total_ns"
+    # (string) Name of the metric tracking allocated bytes (even if freed)
+    totalalloc: "go_expvar.memstats.total_alloc"
+    # (string) Name of the metric tracking number of active go routines
+    goroutinesexists: "go_expvar.goroutines.exists"
+    # (time.Duration) Interval on which metrics are reported
+		reportinterval: "5s"
```

Comment:
I believe this is indicating you have a tab character here that doesn't match the rest of the whitespace.